WordPress/src/main/java/org/wordpress/android/ui/posts/prepublishing/home/viewmodel/slice/AsyncPublishingUiHandler.kt

```diff
@@ -0,0 +1,5 @@
+package org.wordpress.android.ui.posts.prepublishing.home.viewmodel.slice
+
+import org.wordpress.android.ui.posts.prepublishing.home.PrepublishingHomeItemUiState
+
+class AsyncPublishingUiHandler: UiHandler<List<PrepublishingHomeItemUiState>>()
```

Comment:
This is empty for now. The goal is to transfer somehow here the behaviour of closing the bottom sheet and the editor when user taps on "Publish now" button (existing async behaviour).